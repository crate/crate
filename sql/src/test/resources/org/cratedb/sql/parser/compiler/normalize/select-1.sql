SELECT * FROM t WHERE a = 1
