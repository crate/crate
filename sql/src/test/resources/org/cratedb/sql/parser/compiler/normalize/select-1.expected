SELECT * FROM t WHERE (a = 1) AND TRUE